gnd/src/main/java/com/google/android/gnd/persistence/remote/firestore/FirestoreStorageManager.java

```diff
@@ -67,7 +70,8 @@ private StorageReference createReference(String path) {
   @Cold
   @Override
   public Single<Uri> getDownloadUrl(String remoteDestinationPath) {
-    return RxTask.toSingle(() -> createReference(remoteDestinationPath).getDownloadUrl());
+    return RxTask.toSingle(() -> createReference(remoteDestinationPath).getDownloadUrl())
+        .doOnError(e -> Timber.e("StorageException handled and can be ignored"));
```

Comment:
If the message can be ignored, then we should leave the body empty and add this as a comment?

Reply:
The problem is that `StorageException` logs an error in its constructor, making it appear something is failing even when we handle the error and recover from local storage. This misled me to believe the app was crashing because of the `StorageException`. By commenting directly in the log, hoping to save others a similar fate. Adding a comment to explain this here.